server/src/test/java/org/apache/cassandra/sidecar/acl/RoleAuthorizationsCacheTest.java

```diff
@@ -106,7 +118,11 @@ void testCacheSizeAlwaysOne() throws InterruptedException
 
         // New entries fetched during refreshes
         assertThat(cache.getAuthorizations("test_role2").size()).isOne();
+        assertThat(sidecarMetrics.server().cache().rolePermissionsCacheMetrics.snapshot().hitCount()).isZero();
         assertThat(cache.getAll().size()).isOne();
+        assertThat(cache.getAuthorizations("test_role2").size()).isOne();
+        assertThat(sidecarMetrics.server().cache().rolePermissionsCacheMetrics.snapshot().hitCount()).isOne();
+
```

Comment:
Can you also assert on the load success count and potentially introduce a scenario that test the cache miss count?